src/bindings/c/ov/tests/ov_c_api_test.cpp

```diff
@@ -79,6 +82,7 @@ size_t read_image_from_file(const char* img_path, unsigned char *img_data, size_
     return read_size;
 }
 
+#ifdef BUILD_OPENCV_DEPENDENT_TESTS
```

Comment:
Looks like we can remove `mat_2_tensor` function because we don't use it

Reply:
yes, PR12187 does it.